### PR TITLE
Document GHC-58656

### LINKED ISSUE #114: DriverInferredSafeModule -> 58656

### PR DESCRIPTION
**Pull request by BinderDavid**
Fixes #114

### CODE REVIEW

File: message-index/messages/GHC-58656/index.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+---
+title: Module is inferred to be safe
+summary: If a module is inferred as safe, then it should be annotated as safe
+severity: warning
+flag: -Wsafe
+introduced: 9.6.1
+---
+
+GHC implements the SafeHaskell extension which allows programmers to restrict modules to a specific subset which is considered safe.
```
Comment (david-christiansen) [inline message-index/messages/GHC-58656/index.md:9]: Please see #464 for suggested verbiage here!

File: message-index/messages/GHC-58656/inferred-safe/after/InferredSafe.hs [hunk 2]
```diff
@@ -0,0 +1,6 @@
+{-# LANGUAGE Haskell2010, Safe #-}
+{-# OPTIONS_GHC -fwarn-safe #-}
```
Comment (david-christiansen) [inline message-index/messages/GHC-58656/inferred-safe/after/InferredSafe.hs:2]: Why doesn't this match the `flag` field in the metadata for the message?

**Review by david-christiansen — CHANGES_REQUESTED** [review]

File: message-index/messages/GHC-58656/index.md (hunk 1 above)
Comment (BinderDavid) [reply message-index/messages/GHC-58656/index.md]: Done in [e25419e](https://github.com/haskellfoundation/error-message-index/pull/463/commits/e25419e13dc8ad41e11a651af3b1190944894cce)

File: message-index/messages/GHC-58656/inferred-safe/after/InferredSafe.hs (hunk 2 above)
Comment (BinderDavid) [reply message-index/messages/GHC-58656/inferred-safe/after/InferredSafe.hs]: Thanks, that is inconsistent with the other examples, yes. I accidentally used the alternative syntax here. Fixed in [ce410a9](https://github.com/haskellfoundation/error-message-index/pull/463/commits/ce410a9df22bc68aa3fba52978ef04d3fdb1da35)

**Pull request merged**